docs/scripts/TLSClientConfigsForITest.sh

```diff
@@ -36,7 +36,7 @@ echo '
                 return 403;
             }
         }
-'> ~/${FOLDERNAME}/clientCertificateConfigs.txt
+'  | sudo tee ~/${FOLDERNAME}/clientCertificateConfigs.txt
```

Comment:
This sudo should not be necessary as ~/${FOLDERNAME} is in the user's directory.